protostar/configuration_file/configuration_file.py

```diff
@@ -86,6 +86,20 @@ def set_command_names_provider(
     ):
         pass
 
+    @staticmethod
+    def create_appending_cairo_path_suggestion() -> str:
+        return "\n".join(
+            [
+                "You may want to add your new library's path to the 'cairo_path' section of your `protostar.toml` file",
```

Comment:
You can avoid hard-coding the configuration file name by using `self.get_filepath().name`

Reply:
For this, we'd have to have a `ConfigurationFile` object as this method is static. I'm going to replace this message with something like this:
```
"You may want to add your new library's path to the 'cairo_path' section of your configuration file (usually `protostar.toml`).",
```